loops-and-arrays.js

```diff
@@ -22,7 +25,15 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
-const numbers = [2, 4, 5, 29, 38];
+// const numbers = [2, 4, 5, 29, 38];
+//
+// for (let i = 0; i < numbers.length; i++) {
+//     if (numbers[i] % 2 === 0) {
+//         console.log(numbers[i] * 2);
+//     } else if (numbers[i] % 2 !== 0) {
+//         console.log(numbers[i] * 3);
+//     } else console.log(numbers[i])
```

Comment:
Het kan nooit zo zijn dat de eerste én de tweede if niet waar zijn
```suggestion
//     if (numbers[i] % 2 === 0) {
//         console.log(numbers[i] * 2);
//     } else {
//         console.log(numbers[i] * 3);
//     }
```